integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/KeycloakTestResource.java

```diff
@@ -34,8 +36,9 @@
                                 .withHeader("Content-Type", MediaType.APPLICATION_JSON)
                                 .withBody("{\n" +
                                         "    \"jwks_uri\": \"" + server.baseUrl()
-                                        + "/auth/realms/quarkus/protocol/openid-connect/certs\"\n"
-                                        +
+                                        + "/auth/realms/quarkus/protocol/openid-connect/certs\",\n" +
+                                        "    \"introspection_path\": \"" + server.baseUrl()
+                                        + "/auth/realms/quarkus/protocol/openid-connect/token/introspect\"\n" +
```

Comment:
@cemnura Can we have a different `KeycloakTestResource` for the introspection endpoint`, we need to keep it simple for the users :-), i.e, if the users want to test JWT, they don't need to set the introspection path (in most cases) and likewise for the opaque tokens. Please add `KeycloakOpaqueTokenTestResource` instead

Reply:
Sounds great. Added `KeycloakOpaqueTokenTestResource`.